fera_full.py

```
@@ -1,3 +1,189 @@
+    sqliteconn = None
+    try:
+        if(len(sys.argv) == 1): 
+            indexador_fera.import_create_toplevel()            
+            if(global_settings.pathdb == None):
+                return  
+            else:
+                sys.argv.append(str(global_settings.pathdb))
+        global_settings.splash_window = classes_general.Splash_window(global_settings.root)
+        if(len(sys.argv) >= 2): 
+            filename, extension = os.path.splitext(sys.argv[1])
+            if(".pdf" == extension.lower()):
```

Ruff

Yoda condition detected

Rewrite as `extension.lower() == ".pdf"`

(SIM300)